gui/src/components/health/cortx-health-graphical.vue

```diff
@@ -0,0 +1,544 @@
+            d.data.status == "degraded"
+          ) {
+            width = "90px";
+          }
+          return width;
+        })
+        .attr("height", "25px")
+        .attr("rx", 10)
+        .attr("x", 10)
+        .attr("y", 7)
+        .style("fill", function(d: any) {
+          let color;
+          switch(d.data.status) {
+            case "online" :
+              color = "#6ebe49";
+              break;
+            case "offline" :
+              color = "#dc1f2e";
+              break;
+            case "failed" :
+              color = "#dc1f2e";
+              break;
+            case "degraded" :
+              color = "#f7a528";
+              break;
+            case "unknown" :
+              color = "#9e9e9e";
+              break;
+          }
+          return color;
+        });
+      nodeEnter
+        .append("text")
+        .attr("dy", "24px")
+        .attr("x", function(d: any) {
+          let xValue;
+          if (d.data.status == "online" || d.data.status == "offline" || d.data.status == "failed") {
+            xValue = d.children || d._children ? "68px" : "24px";
+          } else if (
+            d.data.status == "unknown" ||
+            d.data.status == "degraded"
+          ) {
+            xValue = d.children || d._children ? "86px" : "23px";
+          }
+          return xValue;
+        })
+        .attr("text-anchor", function(d: any) {
+          return d.children || d._children ? "end" : "start";
+        })
+        .style("fill", "#fff")
+        .style("text-transform", "uppercase")
+        .style("font-size", "12px")
+        .text(function(d: any) {
+          return d.data.status;
+        });
+      nodeEnter
+        .append("text")
+        .attr("x", "10px")
+        .attr("y", "60px")
+        .attr("id", function(d: any) {
+          return "nodeInfoTextID" + d.id + d.depth;
+        })
+        .style("visibility", "hidden")
+        .append("tspan")
+        .style("font-size", "10px")
+        .text(function(d: any) {
+          return "Name: " + d.data.resource;
+        })
+        .append("tspan")
+        .style("font-size", "10px")
+        .attr("x", "10px")
+        .attr("dy", "15px")
+        .text(function(d: any) {
+          return "ID: " + d.data.id;
+        })
+        .append("tspan")
+        .attr("x", "10px")
+        .attr("dy", "15px")
+        .style("font-size", "10px")
+        .text(function(d: any) {
+          let time = new Date(d.data.last_updated_time * 1000);
+          time = moment.default(time).format("DD-MM-YYYY hh:mm A");
+          return "Time: " + time;
+        });
+      nodeEnter
+        .append("image")
+        .attr("class", "node")
+        .attr("xlink:href", require("@/assets/info-icon.svg/"))
+        .attr("x", "115px")
+        .attr("y", "10px")
+        .attr("width", "20px")
+        .attr("height", "20px")
+        .style("cursor", "pointer")
+        .on("click", function(d: any) {
+          if (
```

Comment:
You can get all the required elements before if and simply set the attributes depending on condition

Reply:
Created variables for elements.